taxi/templates/base.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH"
+      crossorigin="anonymous"
+    />
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'css/styles.css' %}" />
+      {%block title%}
+    <title>Taxi Service</title>
+    {%endblock %}
+  </head>
+  <body>
+    <h1>Home page</h1>
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-sm-3">
+          {% block sidebar%} {% include "includes/sidebar.html" %} {% endblock%}

```

Comment:
Ensure consistent spacing in template tags. Consider adding a space after `{% block sidebar%}` and before `{% endblock%}` for better readability.